reports/win32_os.go

```diff
@@ -13,14 +13,12 @@ func GetWin32OS() (Win32OS, error) {
 	o, err := cmd.Output()
 
 	if err != nil {
-		fmt.Println("error here")
+		log.Printf("error here")
```

Comment:
remove this line, and change `return Win32OS{}, err` to
```
return Win32OS{}, errors.Wrap(err, "exec gwmi Win32_OperatingSystem")
```
Or something similar to that.

Reply:
this is mostly resolved in 25b925c036238d9893a52f61b0f814e062e08ea5, however i have a bit or work to do in emulators which i will address in another PR